guides/micronaut-integration-testing/groovy/src/main/groovy/example/micronaut/HelloRepository.groovy

```diff
@@ -0,0 +1,14 @@
+package example.micronaut
+
+import io.micronaut.context.annotation.DefaultImplementation
+
+@DefaultImplementation(HelloRepositoryDefault.class) // <1>
+interface HelloRepository {
+
+    String getHelloInLanguage(String language)
+
+    void putHelloInLanguage(String language, String hello)
+
+    void clearRepository()
```

Comment:
delete this method which is only used for tests. add it to the bean replacement.